### ISSUE TITLE
[DDL] Add Board of Appeals decision letters 

### ISSUE THREAD

**Issue by jacobworrell** [department-of-veterans-affairs/va.gov-team#55071]
## Value Statement
**_As a_** veteran waiting on a board of appeals decision
**_I want to_** be able to see my board of appeal decision letter as soon as the decision is made by The Board
**_So that_** I don't have to wait days for it to arrive in the mail and I can use it to plan my next moves accordingly

---
## Background Context
- Currently veterans have to wait for their Board of Appeals decision letters in snail mail. BVA is eager to surface these letters within the DDL feature. According to our contact at CaseFlow, we would only need to filter by doc ID "27" to expose these letters to end users. That doc type does not seem to be associated with any other documents other than Board of Appeals decision letters. 

## Acceptance Criteria
- [ ] Board of Appeals letters appear in the DDL List View
- [ ] Board of Appeals letters are appropriately distinguished from other Notification Letters are can be clearly identified as what they are
- [ ] Clicking on a download link downloads the correct document type, a Board of Appeals letter pdf
- [ ] Closed board appeals provide a link to the DDL List View

## Designs and Build Notes
- Add here

## Out of scope
- Add here

## Open questions and assumptions
- We can refer to the metadata to populate the list text
- Doc type ID only shows board of appeals letters
- Should we add a separate GA event for this? 

## Tasks
- [ ] _Notify Mobile we're doing this_
- [ ] _Task_
- [ ] _Task_

## Definition of Ready 

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Reviewed and approved by product and/or design

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by jacobworrell** [comment]
Vertically slice this
